pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/Messages.properties

```diff
@@ -142,7 +142,7 @@ ModelValidatorImpl.NestedWhenNoArgs=No additional arguments are allowed for chil
 ModelValidatorImpl.NoNestedWhenAllowed=No children when conditions are allowed for when condition "{0}".
 ModelValidatorImpl.NestedWhenWithoutChildren=Nested when condition "{0}" requires at least one child condition.
 ModelValidatorImpl.NestedWhenWrongChildrenCount=Nested when condition "{0}" requires exactly {1} child condition(s).
-ModelValidatorImpl.BothStagesAndSteps=Only one of "parallel" or "steps" allowed for stage "{0}"
+ModelValidatorImpl.TwoOfStepsStagesParallel=Only one of "parallel", "stages", or "steps" allowed for stage "{0}"
```

Comment:
This is better (compare above)

Note to self: The periods in the nearby lines are probably bugs -- or indications of bugs...